packages/samples/specs/resource-manager/operations/operation-provider/main.tsp

```diff
@@ -48,34 +48,33 @@ model LogAnalyticsCollection is Page<LogAnalyticsOperationResult>;
 interface ProviderOperations {
   /** Operation to get virtual machines for subscription (/subscriptions/{subscriptionId}/providers/Microsoft.ContosoProviderHub/getVmSizes) */
   @get
-  getVmSizes is ArmProviderActionAsync<
-    Response = VmSizeCollection,
+  getVmSizes is ArmProviderActionSync<Response = VmSizeCollection, Scope = SubscriptionActionScope>;
+  /** Operation to post virtual machines for subscription (/subscriptions/{subscriptionId}/providers/Microsoft.ContosoProviderHub/postVmSizes) */
```

Comment:
Can we add an action with customized LroHeader with combined lro header and @usseFinalStateVia to select non-default auzre-async-operation?